jib-core/src/test/java/com/google/cloud/tools/jib/cache/CacheStorageWriterTest.java

```diff
@@ -117,7 +150,7 @@ public void testWriteUncompressed() throws IOException {
 
     CachedLayer cachedLayer = cacheStorageWriter.writeUncompressed(uncompressedLayerBlob, selector);
 
-    verifyCachedLayer(cachedLayer, uncompressedLayerBlob);
+    verifyCachedLayer(cachedLayer, uncompressedLayerBlob, compress(uncompressedLayerBlob));
```

Comment:
The test is about uncompressed layers, but I noticed the verifyCachedLayer actually uses compression for asserts.
What is even more strange is that this test seems dependant on the gzip compression settings: it passes with JDK GZIPOutputStream, but not with common-compress GzipCompressorOutputStream.

Reply:
Seems like a convenience use... @chanseokoh you had a comment in #1678 saying it's a common but unnecessary pattern to "wrap anything in a Blob only for the sake of writing it to an output stream or compute a digest". Do you recall if that convenience shortcut is the reason `verifyCachedLayer` happens to compress the Blob before getting a descrptor out of it?